src/docs/development/frameworks-mobile/react-native.mdx

```diff
@@ -58,11 +58,13 @@ Before taking the following steps, you'll want to make sure that you have an exi
 React Native projects should make use of the [React-Native-Paper](https://callstack.github.io/react-native-paper/index.html) component library. Our Brightlayer UI themes are designed to work with these components. You'll also want to install [react-native-vector-icons](https://www.npmjs.com/package/react-native-vector-icons) to access the Material icons.
 
 ```sh
-npm install --save react-native-paper react-native-vector-icons
+npm install --save react-native-paper@^4.0.0 react-native-vector-icons
 // or
-yarn add react-native-paper react-native-vector-icons
+yarn add react-native-paper@^4.0.0 react-native-vector-icons
 ```
 
+> **NOTE**: react-native-paper@^5.0.0 is currently available, however, we only support RNP v4 at this time. Please be sure to keep peer-dependencies aligned as defined above to avoid potential conflicts.
```

Comment:
avoid acronyms like "RNP", especially when it is mentioned for the first time on this page.

Reply:
Fair point, I made assumptions that there was enough context to decipher what RNP meant.